### DISCUSSION TITLE
Problem with opening xrdml pole figure data file

### DISCUSSION THREAD

**Discussion by Samarendra0101** [mtex-toolbox/mtex#1439] [Ask Anything]
I'm not being able to open my .xrdml pole figure data file in the mtex toolbox. The files were generated using a Panalytical empyrean system. I'm attaching the files as well, kindly find the files in the following onedrive link, can anyone kindly help?

https://iitkgpacin-my.sharepoint.com/:f:/g/personal/samarendra_iitkgp_ac_in/EgQr0VnspUFDt7Zls8o8Q7IB0tVjzQBpirNNqEQAkegIuQ?e=ycSVDB

Samarendra

**Comment by kilir** [comment]
Hi,
```
pf = PoleFigure.load('A30 Texture-pole2-polycap-cradle-dCore-Pixcel.xrdml')
```
works fine for me. Of corse you may additionally want to specify crystalSymmetry and eventually the right Miller for the pf.

Cheers,
Rüdiger

**Reply by Samarendra0101** [reply]
Thanks Rüdiger, for your response. However, for my case it is not working. I'm using matlab 2020b and mtex 5.1.1
Can you confirm which version you are using. I have attached the screenshot of the error code. Kindly have a look.
Samarendra
![file load problem](https://user-images.githubusercontent.com/106327961/170643205-f2df8df7-69d9-42e6-9a87-84a82d238de6.png)

**Reply by kilir** [reply]
Hi,
5.8.1
Cheers,
Rüdiger
